File: 1013-fibonacci-number/fibonacci-number.py
class Solution:
    def fib(self, n: int) -> int:
        arr = [0, 1]
        for i in range(n - 2):
            arr[0], arr[1] = arr[1], arr[0] + arr[1]
        return arr[0] + arr[1] if n > 1 else n

File: 1013-fibonacci-number/fibonacci-number.cpp
class Solution {
public:
    int fib(int n) {
        int arr[2] = {0, 1};
        for (int c = 2; c < n; c++){
            int tmp = arr[0];
            arr[0] = arr[1];
            arr[1] += tmp;
        }
        return n > 1 ? arr[0] + arr[1] : n;
    }
};